parsers/AX.py

```diff
@@ -303,7 +306,7 @@ def fetch_production(country_code='AX', session=None):
         'source': 'kraftnat.aland.fi',
         'datetime': arrow.get(obj['fetchtime']).datetime
     }
-    data['production']['biomass'] = round(obj['unknown'],1)
+    data['production']['biomass'] = None
```

Comment:
Maybe you should now assign `obj['unknown']` to `unknown` as it was removed from `biomass`?

Reply:
We do not know the amount of any type of production except wind and oil based on the source. I can of course make obj['unknown'] equal None and make this change, but is there a point in doing so?